### PR TITLE
chore: replace staging basic example URL with relative URL

### PR DESCRIPTION
**Pull request by ankri**
Currently when netlify is hosting the PR preview the URL for the basic example will link to the production example. This should fix that by providing a relative link.

Would it be possible to configure Netlify in a way, that they not only post the URL of the landing example, but also the basic example in their comment?

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
     staging: {
       LANDING: 'https://craftjs.netlify.com/',
       DOCUMENTATION: 'https://craftjs.netlify.com/r/docs/overview/',
```
Comment (prevwong) [inline config.js:10]: Can we change these too?

**Review by prevwong — CHANGES_REQUESTED** [review]
Thanks!

File: config.js (hunk 1 above)
Comment (ankri) [reply config.js]: Done 👍

**Review by prevwong — APPROVED** [review]

**Comment by prevwong** [comment]
Closing since this is not needed anymore since we refactored our docusaurus site structure

**Pull request closed without merging**